docs/reborn/target-architecture/explorer.html

```
@@ -0,0 +1,441 @@
+<title>IronClaw — Target Architecture Explorer</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)